MathPortal/MathPortal/Source/UserInterface/TabBarViewController.swift

```diff
@@ -15,25 +15,28 @@ class TabBarViewController: UIViewController {
     
     @IBOutlet private var firstButton: TabBarButton?
     @IBOutlet private var secondButton: TabBarButton?
-    @IBOutlet var thirdButton: TabBarButton?
+    @IBOutlet private var thirdButton: TabBarButton?
+    @IBOutlet private var fourthButton: TabBarButton?
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
         firstButton?.addTarget(self, action: #selector(firstButtonPressed), for: .touchUpInside)
         secondButton?.addTarget(self, action: #selector(secondButtonPressed), for: .touchUpInside)
         thirdButton?.addTarget(self, action: #selector(thirdButtonPressed), for: .touchUpInside)
+        fourthButton?.addTarget(self, action: #selector(fourthButtonPressed), for: .touchUpInside)
```

Comment:
If these are in storyboard then why are targets not as well?

Reply:
The buttons are actually CustomViews that contain label and button. Will change the naming to firstButtonView and TabBarButtonView 😄